chain/events/filter/event.go

```diff
@@ -432,7 +431,7 @@ func (m *EventFilterManager) Remove(ctx context.Context, id types.FilterID) erro
 	return nil
 }
 
-func (m *EventFilterManager) loadExecutedMessages(ctx context.Context, msgTs, rctTs *types.TipSet) ([]executedMessage, error) {
+func (m *EventFilterManager) LoadExecutedMessages(ctx context.Context, msgTs, rctTs *types.TipSet) ([]executedMessage, error) {
```

Comment:
the lint on this line is reasonable, I think you can export `ExecutedMessage` as it is, it has accessors on the private properties anyway